src/main/java/de/retest/recheck/auth/RetestAuthentication.java

```diff
@@ -37,7 +37,7 @@
 	public static final String AUTH_SERVER_PROPERTY = "de.retest.auth.server";
 	private static final String AUTH_SERVER_PROPERTY_DEFAULT = "https://sso.prod.cloud.retest.org/auth";
 	public static final String RESOURCE_PROPERTY = "de.retest.auth.resource";
-	private static final String RESOURCE_PROPERTY_DEFAULT = "review";
+	private static final String RESOURCE_PROPERTY_DEFAULT = "marvin";
```

Comment:
Do we make sure in downstream projects to change that (i.e. change the default)?

Reply:
This is/can be overwritten (even in the `Rehub` class).